accounts/test_kvdb.go

```diff
@@ -0,0 +1,30 @@
+package accounts
```

Comment:
Nit: the commit message contains info regarding build flags logic, which isn't present in the actual commit (which i suspect will be added later?).

Reply:
yes they will be added later. included them in the commit message so there is some context for this commit